lib/util/regex.ts

```diff
@@ -98,3 +98,12 @@ export function configRegexPredicate(
   }
   return null;
 }
+
+const UUIDRegex = regEx(
+  /^\{[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\}$/,
+  'i'
+);
+
+export function isUUID(input: string): boolean {
```

Comment:
please add a test to `./regex.spec.ts`, so this function is covered when only running tests inside the utils folder.

Reply:
Added one, it is pretty basic but should work. I tried to match convention here as much as possible.